osmaxx-py/excerptconverter/converter_helper.py

```diff
@@ -0,0 +1,49 @@
+from django.contrib import messages
+from django.core.mail import send_mail
+from django.utils.translation import ugettext_lazy as _
+
+import stored_messages
+
+from osmaxx.excerptexport import models
+
+from excerptconverter.baseexcerptconverter import BaseExcerptConverter
+
+class ConverterHelper:
+    def __init__(self, extraction_order):
+        self.extraction_order = extraction_order
+        self.user = extraction_order.orderer
+
+    def file_conversion_finished(self):
+        if self.extraction_order.output_files.count() >= len(self.extraction_order.extraction_formats):
+            self.inform_user(
+                messages.SUCCESS,
+                _('The extraction of the order "%(order_id)s" has been finished.') % {
+                    'order_id': self.extraction_order.id
+                },
+                True
```

Comment:
Always use named arguments for boolean parameters. See http://ariya.ofilabs.com/2011/08/hall-of-api-shame-boolean-trap.html